src/lib/components/Polling.svelte

```diff
@@ -18,6 +19,10 @@
         setTimeout(poll, currentInterval)
     }
 
+    async function updateTypingIndicators() {
+        UIStore.updateTypingIndicators()
```

Comment:
ReferenceError: UIStore is not defined


<img width="1067" alt="Captura de ecrã 2024-08-07, às 12 56 16" src="https://github.com/user-attachments/assets/5db0ad64-5dfe-4758-94f8-673477806731">